File: src/app/admin/XXXXX.Admin.Core/Interfaces/IRoleAdminPermissionService.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;

using XXXXX.Admin.Core.ViewModels;

namespace XXXXX.Admin.Core.Interfaces
{
    public interface IRoleAdminPermissionService
    {
        Task<IEnumerable<PermissionAdminInfosViewModel>> GetMany(Guid appId, Guid actorId, Guid roleAdminId);
        Task Update(Guid appId, Guid actorId, Guid roleAdminId, List<Guid> payload);
    }
}

File: src/app/admin/XXXXX.Admin.UI/src/interfaces/iroleAdminPermissionService.ts
import { PermissionAdminInfos, PermissionAdminFilter } from "@/domain/models/permissionAdmins";

export interface IRoleAdminPermissionService {
    getMany(roleAdminId: string, filter?: PermissionAdminFilter): Promise<PermissionAdminInfos[]>;
    update(roleAdminId: string, payload: string[]): Promise<void>
}